<ui version="4.0" >
 <class>OutputConfigBase</class>
 <widget class="QWidget" name="OutputConfigBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>213</width>
    <height>87</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Output Config</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>10</number>
   </property>
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QCheckBox" name="activeCheck" >
     <property name="text" >
      <string>Active</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Size:</string>
       </property>
       <property name="buddy" >
        <cstring>sizeCombo</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="sizeCombo" />
     </item>
     <item>
      <widget class="QLabel" name="rateLabel" >
       <property name="text" >
        <string>Refresh:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="refreshCombo" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="orientationLabel" >
       <property name="text" >
        <string>Orientation:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="orientationCombo" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
